creator-node/src/fileManager.js

```diff
@@ -187,7 +187,7 @@ async function saveFileForMultihashToFS (req, multihash, expectedStoragePath, ga
         decisionTree.push({ stage: 'About to retrieve file from local ipfs node with get', vals: multihash, time: Date.now() })
         // ipfsGet returns a BufferListStream object which is not a buffer
         // not compatible into writeFile directly, but it can be streamed to a file
-        let fileBL = await Utils.ipfsGet(multihash, req, 5000)
+        let fileBL = await Utils.ipfsGet(multihash, req, 1000)
```

Comment:
are we sure we want to reduce timeout? not sure how long this takes for a massive downloadable file

Reply:
so the ipfsCat is 1000 so I just made this consistent with that. also i remember testing this before and that timeout is time to first byte, not entire file transfer. so as long as the transfer starts within the timeout it's okay

Reply:
ohhh true, all good then

Reply:
wait but that is no longer the case with your new js setTimeout, bc that will abort past just the first byte

Reply:
i can make the js timeout 2x the ipfs timeout so we give it a decent buffer?

Reply:
yee lets do this

Reply:
fixed